cli/src/main/java/com/devonfw/tools/ide/process/ProcessContextImpl.java

```diff
@@ -313,22 +316,34 @@ private String addExecutable(String exec, List<String> args) {
   private void performLogOnError(ProcessResult result, int exitCode, String interpreter) {
 
     if (!result.isSuccessful() && (this.errorHandling != ProcessErrorHandling.NONE)) {
-      String message = createCommandMessage(interpreter, " failed with exit code " + exitCode + "!");
+      IdeSubLogger subLogger = this.context.error();
+      IdeLogLevel ideLogLevel;
+      String message = createCommandMessage(interpreter, "\nfailed with exit code " + exitCode + "!");
+
       if (this.errorHandling == ProcessErrorHandling.THROW_CLI) {
+        subLogger.log(message);
```

Comment:
if we still hardcode the log-level to `error` why then hiding this information making the code in-transparent?
```suggestion
        this.context.error(message);
```

Reply:
Adjusted.